tests/functional/TestUpgrade/test_upgrade.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2018 The heketi Authors
+#
+# This file is licensed to you under your choice of the GNU Lesser
+# General Public License, version 3 or any later version (LGPLv3 or
+# later), as published by the Free Software Foundation,
```

Comment:
I guess you copied the license part from python api file. Heketi license is currentyl on GPL 3 or LGPLv3 for core code( api server, tests etc). Only the API is dual licensed under Apache.

Reply:
Yes, that is exactly what I did. What do you think it should be?

Reply:
As in any non-api go file, e.g:

> // This file is licensed to you under your choice of the GNU Lesser
> // General Public License, version 3 or any later version (LGPLv3 or
> // later), or the GNU General Public License, version 2 (GPLv2), in all
> // cases as published by the Free Software Foundation.